library/gcm.c

```diff
@@ -42,38 +42,73 @@
 #if !defined(MBEDTLS_GCM_ALT)
 
 /* Used to select the acceleration mechanism */
-#define MBEDTLS_GCM_ACC_SMALLTABLE  0
-#define MBEDTLS_GCM_ACC_LARGETABLE  1
-#define MBEDTLS_GCM_ACC_AESNI       2
-#define MBEDTLS_GCM_ACC_AESCE       3
-
-/*
- * Initialize a context
- */
-void mbedtls_gcm_init(mbedtls_gcm_context *ctx)
-{
-    memset(ctx, 0, sizeof(mbedtls_gcm_context));
-}
+#define MBEDTLS_GCM_ACC_UNSET       0
+#define MBEDTLS_GCM_ACC_SMALLTABLE  1
+#define MBEDTLS_GCM_ACC_LARGETABLE  2
```

Comment:
If we're touching these lines, can we put an underscore `_` before `TABLE` to match the names of the config parameters (`MBEDTLS_AES_FEWER_TABLES` and `MBEDTLS_GCM_LARGE_TABLE`)